house/migrations/0004_alter_house_children_alter_house_house_id_and_more.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.0.3 on 2022-03-31 10:42
+
+import django.core.validators
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('house', '0003_delete_job_alter_city_id_alter_country_id_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='house',
+            name='children',
+            field=models.IntegerField(blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)]),
+        ),
+        migrations.AlterField(
+            model_name='house',
+            name='house_id',
+            field=models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False),
+        ),
+        migrations.AlterField(
+            model_name='house',
+            name='income',
+            field=models.IntegerField(blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)]),
+        ),
+    ]
```

Comment:
What is that code doing? do we need that?

Reply:
We do. This file was created as part of the migrations and is required for the delete-cascade functionality